go_test.go

```diff
@@ -12,13 +12,25 @@ func TestGoVersion(t *testing.T) {
 	}
 
 	acceptable := []string{
-		"devel", "go1.0", "go1.1", "go1.2", "go1.3",
-		"go1.4", "go1.4.1", "go1.4.2", "go1.4.3",
-		"go1.5", "go1.5.1", "go1.5.2", "go1.5.3", "go1.5.4",
-		"go1.6", "go1.6.1", "go1.6.2", "go1.6.3", "go1.6.4",
-		"go1.7", "go1.7.1", "go1.7.2", "go1.7.3", "go1.7.4", "go1.7.5", "go1.7.6",
-		"go1.8", "go1.8.1", "go1.8.2", "go1.8.3", "go1.8.4",
-		"go1.9", "go1.9.1", "go1.9.2",
+		"devel", "go1.0",
+		"go1.1",
+		"go1.2",
+		"go1.3",
+		"go1.4",
+		"go1.5",
+		"go1.6",
+		"go1.7",
+		"go1.8",
+		"go1.9",
+		"go1.10",
+		"go1.11",
+		"go1.12",
+		"go1.13",
+		"go1.14",
+		"go1.15",
+		"go1.16",
+		"go1.17",
+		"go1.18",
```

Comment:
This would be very tedious to maintain if we kept track of 10+ patch versions per minor version